department-of-veterans-affairs/caseflow
# Intake | Prevent Re-refilings

![screen shot 2018-01-10 at 8 23 06 pm](https://user-images.githubusercontent.com/1596259/34803945-17b3382e-f644-11e7-8f3c-4aff890afdf7.png)

@abbyraskinUSDS can you confirm/fix the copy?
**title:** Selection form already processed in Caseflow',
**body:**  Caseflow does not currently support the processing of the second, third, etc. Selection form for a veteran. Please contact Caseflow support for instructions on what to do.

completes all AC of the issue.

## To test
- Complete a ramp refiling intake. Attempt to start another one for the same veteran. See that 

## Notes
Trusted just a unit test here to get the job done.

connects #4271
